### DISCUSSION TITLE
Use freeplane for business purpose

### DISCUSSION THREAD

**Discussion by adas2327** [freeplane/docs#150] [General]
Can I do it under an open license? Didnt find any specific documentation on restriction of usage. I apologize if I missed anything obvious.

**Comment by macmarrum** [comment]
You can find lincense information in Freeplane `Help -> About`.

You can also have a look at https://en.wikipedia.org/wiki/GNU_General_Public_License#Use_of_licensed_software

If you have any further questions, please post your questions in https://github.com/freeplane/freeplane/discussions. This forum is focused on Freeplane documentation.
